api/views/__init__.py

```diff
@@ -0,0 +1,187 @@
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.db import IntegrityError
+
+from django_filters import rest_framework as filters
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+from waffle import get_waffle_flag_model
+from waffle.models import Switch, Sample
+from rest_framework import (
+    decorators,
+    permissions,
+    response,
+    viewsets,
+    exceptions,
+)
+
+from privaterelay.settings import (
+    BASKET_ORIGIN,
+    FXA_BASE_ORIGIN,
+    GOOGLE_ANALYTICS_ID,
+    PREMIUM_PROD_ID,
+    PHONE_PROD_ID,
+)
+from privaterelay.utils import get_premium_countries_info_from_request
+
+from emails.models import (
+    CannotMakeAddressException,
+    DomainAddress,
+    Profile,
+    RelayAddress,
+)
+
+
+from ..exceptions import ConflictError
+from ..permissions import IsOwner
+from ..serializers import (
+    DomainAddressSerializer,
+    ProfileSerializer,
+    RelayAddressSerializer,
+    UserSerializer,
+)
+
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Relay API",
+        default_version="v1",
+        description="API endpints for Relay back-end",
+        contact=openapi.Contact(email="lcrouch+relayapi@mozilla.com"),
+    ),
+    public=settings.DEBUG,
+    permission_classes=[permissions.AllowAny],
+)
+
+
+class SaveToRequestUser:
+    def perform_create(self, serializer):
+        serializer.save(user=self.request.user)
+
+
+class RelayAddressFilter(filters.FilterSet):
+    used_on = filters.CharFilter(field_name="used_on", lookup_expr="icontains")
+
+    class Meta:
+        model = RelayAddress
+        fields = [
+            "enabled",
+            "description",
+            "generated_for",
+            "block_list_emails",
+            "used_on",
+            # read-only
+            "id",
+            "address",
+            "domain",
+            "created_at",
+            "last_modified_at",
+            "last_used_at",
+            "num_forwarded",
+            "num_blocked",
+            "num_spam",
+        ]
+
+
+class RelayAddressViewSet(SaveToRequestUser, viewsets.ModelViewSet):
```

Comment:
Nit / Future Work: I'd prefer this stuff in `api/views/emails.py` rather than `api/views/__init__.py`

Reply:
Yeah, I want to come back thru and add a bunch of docs to the other API endpoints too like I did with the phones endpoints.